config/initializers/lograge.rb

```diff
@@ -0,0 +1,18 @@
+Rails.application.configure do
+  config.lograge.enabled = true
+
+  #this adds some data to the log that's helpful for graylog
+  config.lograge.formatter = Lograge::Formatters::Graylog2.new
+  if Rails.env.production?
+    config.log_tags = [ :remote_ip ]
+  end
+  config.lograge.base_controller_class = 'ActionController::API'
+  config.lograge.custom_options = lambda do |event|
+    exceptions = %w(controller action format id)
+    now = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+    {
+      start_time: (now - event.time).seconds.ago,
```

Comment:
Lets make this timestamp formatted like the one in brackets (i.e. "2019-06-06T22:..." instead of "Tue, 06 Aug...")

Actually... since the one in brackets is already prepended to request logs (and appears to be the same timestamp) I think we can remove this?